### ISSUE TITLE
Calibration "succeeds", nothing follows custom curves

### ISSUE THREAD

**Issue by PoisonVendor** [Rem0o/FanControl.Releases#2412]
FanControl runs but with the assisted setup out of the way, the auto-calibrate/pairing function fails to assign correct sensors / controls seemingly at random, and also refuses to follow my custom curves. It seems incapable of controlling the GPU. It also occasionally elects to pick a fan to peg to max, particularly the Pump Fan which I have assigned to an exhaust fan as seen below.

![image](https://github.com/Rem0o/FanControl.Releases/assets/164973658/42b55d58-ece5-465e-93f9-966f9ff2ae3e)

(naming conventions are screwed up because I've been poking at them to get them to work)

No log file

XFX Radeon 7900XT
MSI PRO B650-P Wifi ATX

Hope this helps.

**Comment by Rem0o** [comment]
For fan control to work properly, it needs to be the only one with hands on the steering wheel.  From your description, seems like you got something else messing with your fans.   You got an MSI mobo, do you got MSI software and what kind of settings do you got in the BIOS for your fan headers? Do you got curves in there?

**Comment by PoisonVendor** [comment]
Nope, I turned off anything else that could take control of my fans when I started FanControlling.

**Comment by Skullywag** [comment]
Adding that my 4060 is doing the same thing, i was on v173 and everything was running fine, finally updated thinking its probably worth it ran that calibration thing and now my gpu doesnt follow the curve ive set at all.
